docs/reference/filter_field_definition.rst

```diff
@@ -66,8 +66,27 @@ Example
         }
     }
 
-doctrine_orm_model_autocomplete
--------------------------------
+ArrayFilter
+-----------
+
+This filter is made for filtering on values saved in databases as serialized arrays of strings with the
+``@ORM\Column(type="array")`` annotation. It is recommended to use another table and ``OneToMany`` relations
+if you want to make complex ``SQL`` queries or if your table is too big and you get performance issues but
+this filter can provide some basic queries::
+
+    protected function configureDatagridFilters(DatagridMapper $datagridMapper): void
+    {
+        $datagridMapper
+            ->add('keywords', ArrayFilter::class);
+    }
```

Comment:
Please give some examples how to fill the filter. What are correct user inputs

Reply:
I updated the example, is it ok for you now ?